desktop/core/src/desktop/js/sql/reference/flink/udfReference.ts

```diff
@@ -476,6 +476,6 @@ export const UDF_CATEGORIES: UdfCategory[] = [
   { name: I18n('Aggregate'), isAggregate: true, functions: AGGREGATE_FUNCTIONS },
   { name: I18n('Analytic'), isAnalytic: true, functions: ANALYTIC_FUNCTIONS },
   { name: I18n('Date'), functions: DATE_FUNCTIONS },
-  { name: I18n('Misc'), functions: AUXILIARY_FUNCTIONS },
+  { name: I18n('Window Functions'), functions: WINDOW_FUNCTIONS },
   { name: I18n('String'), functions: STRING_FUNCTIONS }
```

Comment:
BTW: are the other functions valid? (maybe we should trim the list/remove the ones probably copy pasted from Hive)

Reply:
@romainr  , the Other functions like Date, analytic , Aggregate are valid in flink too , i have verified it in here : https://ci.apache.org/projects/flink/flink-docs-release-1.12/dev/table/functions/systemFunctions.html